src/rantanplan/core.py

```diff
@@ -266,7 +298,7 @@ def spacy_tag_to_dict(tag):
 
 def get_word_stress(word, pos, tag):
     """
-    Gets a list of syllables from a word and creates a list with syllabified word and stressed syllable index
+    Gets a list of syllables from a ord and creates a list with syllabified word and stressed syllable index
```

Comment:
What's an `ord`?

Reply:
he mallest lement hat an e ttered n solation ith bjective r ractical eaning

Reply:
fixed

Reply:
That previous message is pretty hard to read.